main.go

```diff
@@ -5,9 +5,10 @@ import (
 	"log"
 	"os"
 	"path"
+	"strconv"
 	"strings"
 
-	github "github.com/pulumi/pulumi-github/sdk/v5/go/github"
+	github "github.com/pulumi/pulumi-github/sdk/v6/go/github"
```

Comment:
We should do the v6 bump in its own PR in case there's any breaking changes.

Reply:
#139